keys.py

```diff
@@ -93,7 +94,7 @@ def return_key_handler(self):
         if self.lines:
             self.selected = self.lines[self.y - 1 if self.y else 0]
             write_to_clipboard(self.selected)
-
+        self.search()
```

Comment:
Oh ok.. got it.. if the second last query has less 3 lines, then it will clear just 3 lines. We need to save the very last search. Is that correct @hmohit